CSS/display_articles_css.css

```diff
@@ -50,63 +50,109 @@ Media query:
     -> the width of the div is redefined as well
  */
 
+
+
 #image_of_article_in_display {
-    height: 260px;
-    max-height: 260px;
+    height: 30vh;
+    max-height: 150px;
 }
 
-@media(max-width: 1070px) {
+
+@media(max-width: 505px) {
```

Comment:
this block will always override the defaults

Reply:
Deleted.